adaptive_components/lib/src/adaptive_column.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+import 'package:flutter/material.dart';
+import 'adaptive_container.dart';
+
+class AdaptiveColumn extends StatelessWidget {
+  const AdaptiveColumn({
+    this.gutter,
+    this.margin,
+    this.columns,
+    @required this.children,
+  }) : assert(children != null);
+
+  final double margin;
+  final double gutter;
+  final AdaptiveConstraintsColumn columns;
+  final List<AdaptiveContaine> children;
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(
+      builder: (BuildContext context, BoxConstraints constraints) {
+        BreakpointSystemEntry _entry = getBreakpointEntry(context);
+        final double _margin = margin ?? _entry.margin;
+        final double _gutter = gutter ?? _entry.gutter;
+        final int _totalColumns =
+            columns?.getAdaptiveConstraintsColumn(context) == 0
+                ? columns.getAdaptiveConstraintsColumn(context)
+                : _entry.columns;
+
+        return Container(
+          color: Colors.grey,
+          width: MediaQuery.of(context).size.width - (_margin * 2),
+          margin: EdgeInsets.symmetric(horizontal: _margin),
+          constraints: BoxConstraints(
+            minWidth: _entry.adaptiveWindowType.widthRangeValues.start,
+            maxWidth: _entry.adaptiveWindowType.widthRangeValues.end,
+          ),
+          child: Wrap(
+            runSpacing: 8.0,
+            children: () {
+              int currentColumns = 0;
+              int totalGutters = 0;
+              List<Widget> childs = [];
```

Comment:
Rename to children, and then use `this.children` to access the class property.
```suggestion
              List<Widget> children = [];
```

Reply:
Done